src/TodoList/hooks/useWindowSize.ts

```
@@ -0,0 +1,30 @@
+import { useEffect, useState } from 'react';
+
+const isBrowser = typeof window !== 'undefined';
+
+const useWindowSize = (
+  initialWidth = Infinity,
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)